packages/@dcl/ecs/src/initialization.ts

```diff
@@ -37,3 +37,17 @@ if (dcl) {
       .finally(() => engine.update(dt))
   })
 }
+
```

Comment:
can we move this to the `systems` folder ?
such like we have the crdt system inside the engine

Reply:
also, call this fn in the `engine.update()` fn instead of adding to the systems container.
So its explicit that we are killing all the results after all the update ticks